v21.1/set-locality.md

```diff
@@ -96,7 +96,7 @@ To add a new row to a regional by row table, you must choose one of the followin
     INSERT INTO {table} (crdb_region, ...) VALUES ('us-east-1', ...);
     ~~~
 
-This is necessary because every row in a regional by row table must have a home region.
+This is necessary because every row in a regional by row table must have a home region.  Note that until you set a home region for a row in a regional by row table, it defaults to the value of the primary region.
```

Comment:
not entirely true; it defaults to the `gateway_region()`, which if the gateway_region does not belong to the multi-region database then defaults to the PRIMARY REGION (hence, `DEFAULT default_to_primary_region(gateway_region())`).

Reply:
Hi Oliver!

Thanks for the correction.  I have updated it using approx. your words as follows.  PTAL

> Note that until you set a home region for a row in a regional by row table, it defaults to the value returned by the built-in function `gateway_region()`. If the value returned by `gateway_region()` does not belong to the multi-region database the table is a part of, the home region defaults to the database's primary region.

Reply:
I would replace `Note that until you set a home region for a row` with `If you do not set a home region for a row`, because for me "Note that until" implies for me implies you should change this later, but that's a nit and me reading too far into it...

Reply:
I like your suggested language better - more clear and direct!  Updated it to that.

Thanks again Oliver! Passing on to docs team reviewer